gentelella/app/js/loadTopo.js

```diff
@@ -98,6 +100,7 @@ function sendTopo() {
 
     function onLoad(e) {
         if (req.status >= 200 && req.status <= 299) {
```

Comment:
what is the real code returned by the resquest? why do we need to filter of the 200+ range?